docs/userguide.md

```diff
@@ -305,6 +305,15 @@ To cleanup the example simply delete the namespace:
 kubectl delete ns test-namespace
 ```
 
+### Troulbeshooting
+
+If you found the federated resources were not propagated to the member clusters, you can always
+use the following command to get some failure reason.
+
+```bash
+kubectl describe <federate resource type> -n test-namespace
```

Comment:
What does `<federate resource type>` refers to?  And what information should the user expect to see?  I was under the impression that determining propagation errors would require inspecting the log of the controller manager, but maybe I'm forgetting something.

Reply:
Done, I updated `<federate resource type>` to `<federate CRD>`, also add a step for how to check controller log.